### DISCUSSION TITLE
Print Requested QSLs: Choose type of QSL sent

### DISCUSSION THREAD

**Discussion by m0urs** [magicbug/Cloudlog#1568] [Ideas / Feature Requests]
I am using the "Print Requested QSLs" feature not only for marking QSLs which I do need to send via bureau, but also for sending out electronic QSLs via mail or direct QSLs.

Currently if I am pushing the button "Mark requested QSLs as sent", all records are updated with "sent via bureau" and I need to manually re-edit all QSOs to match the real status.

I would like to have an option to choose which type of QSL I have sent before I press that button. I do not need that for every single QSO but it would be sufficient to have it for all QSOs currently in the queue.

**Comment by dg9vh** [comment]
@phl0 how would this be documented in your development-branch of the email-Qsl? This is also a subject in this field.

**Comment by AndreasK79** [comment]
@m0urs it seems to that when #1557 is done, this should fit your needs.

**Reply by m0urs** [reply]
Thanks, yes, that looks really great and I am looking forward to test that feature soon!
